Rakefile

```diff
@@ -147,11 +144,12 @@ task 'gem:native' do
     # Truncate grpc_c.*.ruby files because they're for Windows only.
     File.truncate('grpc_c.32.ruby', 0)
     File.truncate('grpc_c.64.ruby', 0)
-    ['x86_64-linux', 'x86-linux'].each do |plat|
-      run_rake_compiler plat, <<-EOT
+    ['x86_64-linux', 'x86-linux', 'x86_64-darwin', 'arm64-darwin'].each do |plat|
```

Comment:
I believe that we we run `rake gem:native` on macos hosts (for the mac package build), this will also build the `'x86_64-darwin` packages on macos - so I want to say we're building this packages twice: once on linux via docker and again on macos.

Since this set of cross-compilation rake tasks is only needed for cross-compilation from linux, can we run this conditionally, only under linux?

I think that will take care of the macos package build timeout that happened in the last package build link posted.

Reply:
Whoops! Sorry yes you're right. I've kicked off another package build:  https://fusion2.corp.google.com/invocations/684801ca-5bca-4ae1-90b7-8f296300ef00 (the previous timeout may just be a flake):

Reply:
Thanks! I can't access that link, but please let me know if it goes green?

Reply:
https://fusion2.corp.google.com/invocations/684801ca-5bca-4ae1-90b7-8f296300ef00 is green!